src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -2354,4 +2354,31 @@ Uninstall the following packages: {0}?</value>
   <data name="MessageSamePackageDiffVersInStdLib" xml:space="preserve">
     <value>A different version of the package {1} is already installed as part of the Standard Library. {0} cannot override packages in the Standard Library location.</value>
   </data>
-</root>
+  <data name="PreferencesViewVisualSettingsGeoScaling" xml:space="preserve">
+    <value>Geometry Scaling</value>
+    <comment>Expander Header Name</comment>
+  </data>
+  <data name="PreferencesViewVisualSettingsGroupStyles" xml:space="preserve">
+    <value>Group Styles</value>
+    <comment>Expander Header Name</comment>
+  </data>
+  <data name="PreferencesViewVisualSettingsRenderPrecision" xml:space="preserve">
+    <value>Render Precision</value>
+    <comment>Expander Header Name</comment>
+  </data>
+  <data name="PreferencesViewStyleWarning" xml:space="preserve">
+    <value>Style name already in use</value>
+  </data>
+  <data name="PreferencesViewVisualSettingsDisplaySettings" xml:space="preserve">
+    <value>Display Settings</value>
+    <comment>Expander Header Name</comment>
+  </data>
+  <data name="PreferencesViewVisualSettingShowEdges" xml:space="preserve">
+    <value>Show Edges</value>
```

Comment:
I think we already have `Show Edges` defined as `DynamoViewSettingsMenuShowEdges`, can you remove the other resource string or reuse by renaming that one to match?

Reply:
@QilongTang I removed the existing DynamoViewSettingsMenuShowEdges resource in commit: https://github.com/DynamoDS/Dynamo/pull/11636/commits/0fb11cde9886451ae02192f52dfc92552696ce67